Build/scripts/add_msbuild_path.cmd

```diff
@@ -57,7 +60,7 @@ if exist "%MSBUILD_PATH%\msbuild.exe" (
     goto :MSBuildFound
 )
 
-echo Dev14 not found, trying Dev12...
+echo Dev14 not found, trying to locate Dev12...

```

Comment:
Why do we still bother trying to find dev12?

Reply:
Good point.